packages/sources/tradingeconomics-test/src/endpoint/price.ts

```diff
@@ -0,0 +1,88 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { EndpointTypes } from './price-router'
+
+export interface ProviderResponseBody {
+  Symbol: string
+  Ticker: string
+  Name: string
+  Country: string
+  Date: string
+  Type: string
+  decimals: number
+  state: string
+  Last: number
+  Close: number
+  CloseDate: string
+  MarketCap: number | null
+  URL: string
+  Importance: number
+  DailyChange: number
+  DailyPercentualChange: number
+  WeeklyChange: number
+  WeeklyPercentualChange: number
+  MonthlyChange: number
+  MonthlyPercentualChange: number
+  YearlyChange: number
+  YearlyPercentualChange: number
+  YTDChange: number
+  YTDPercentualChange: number
+  day_high: number
+  day_low: number
+  yesterday: number
+  lastWeek: number
+  lastMonth: number
+  lastYear: number
+  startYear: number
+  ISIN: string | null
+  frequency: string
+  LastUpdate: string
+}
+
+type HttpEndpointTypes = EndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ProviderResponseBody[]
+  }
+}
+
+export const httpTransport = new HttpTransport<HttpEndpointTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      const symbol = `${param.base}${param.quote}:CUR`
+      return {
+        params: [param],
+        request: {
+          baseURL: config.API_ENDPOINT,
+          url: `/symbol/${symbol}`,
+          params: {
+            c: `${config.API_CLIENT_KEY}:${config.API_CLIENT_SECRET}`,
+            f: `json`,
+          },
+        },
+      }
+    })
+  },
+  parseResponse: (params, res) => {
+    const data = res.data[0]
+    return params.map((param) => {
+      if (!res.data || !data || data.Last === undefined) {
+        return {
+          params: param,
+          response: {
+            statusCode: 502,
+            errorMessage: `Tradingeconomics provided no data for ${JSON.stringify(param)}`,
+          },
+        }
+      }
+      return {
+        params: param,
+        response: {
+          data: {
+            result: data.Last,
+          },
+          result: data.Last,
+        },
```

Comment:
If the response includes a timestamp for the data, could you also include `providerIndicatedTimeUnixMs` here?